docs/specs/future-of-workload.md

```diff
@@ -0,0 +1,43 @@
+
+Also, there is no real need of including the Aspire.Hosting package as part of the workload, as there is no guarantee that this is the version that the AppHost project will want to use, therefore we would also remove this package from the workload. In the end, the workload will only contain the Aspire.Hosting.SDK and Aspire.ProjectTemplates components, which are the ones that can be truly considered as tooling components, and in both cases you would want to always be in the latest version. These two are also not tightly coupled to the version of the Aspire.Hosting package that the AppHost project references, so they can be updated independently of the AppHost.
+
+This is the proposed new design:
+
+![Proposal](workload-proposed-state.png "Proposal")
+
+### Pros
+
+This proposal has many benefits which will solve the issues we currently have with the workload:
+
+- The AppHost project will have full control over the version of the DCP and Dashboard that it wants to use.
+- Customers can have multiple AppHost projects that reference different versions of Aspire.Hosting without any issues.
+- We can update the DCP and Dashboard independently of the workload, which will allow us to ship updates to these components more frequently and with less risk.
+- Customers can stay in the latest version of the workload without worrying about compatibility issues with the AppHost project.
+
+### Cons
+
+- The main drawback of removing DCP and Dashboard from the workload, is the potential impacts it may have with restore times and local package caches. DCP especially is not small in size (~150mb) and it is also a native executable (written in Go), so we currently have separate packages each carrying the executable for each individual platform. One thing we were getting for free with DCP being part of the workload, is that the SDK authoring would automatically only install the DCP package for the platform that the SDK is being installed on. NuGet doesn't have great support today for platform-specific dependencies, which means that with the above proposal the Aspire.Hosting.AppHost package would depend on all of the DCP packages for all platforms, which would increase the size of the things that need to be restored, as well as the time it takes to do it.
+- Additionally, workloads also have a way of keeping a reference count on workload packs, so it is able to clean them up when workloads don't reference it any more (for example, if you update your workload, then the old DCP package will be cleaned up automatically). This doesn't happen with regular NuGet packages, where customers are responsible for cleaning up their NuGet package caches themselves.
```

Comment:
I'm not super concerned about this as managing the package restore location is already a scenario users have to manage.

Reply:
Fair enough, I still wanted to mention it because technically it is a drawback, but I agree with you that is not a major concern and it still seems like the right move.

Reply:
It's definitely a concern we should bring back up with the nuget folks (cough cough @JonDouglas ) because it's an eternal pain point lol. And, in the future, if Aspire is used by people who aren't primarily .NET devs they might not know to clear it. But agree not a blocker, especially for now!

Reply:
Yes, please discuss your NuGet needs for this work long term with us. Whether that's moving away from workloads or not, we'd love to help.